### PR TITLE
Add support for i925

### PR DESCRIPTION
**Pull request by nicklovell23**

### CODE REVIEW

File: boardconfig/BoardConfig.mk [hunk 1]
```diff
@@ -417,6 +417,16 @@ else ifneq ($(filter $(TARGET_PRODUCT),cm_lt03wifi cm_lt03wifiue),)
     BRIGHTNESS_SYS_FILE := "/sys/class/backlight/panel/brightness"
     BOARD_USE_B_SLOT_PROTOCOL := true
 
+#Galaxy Note 10.1 - i925
+else ifneq ($(TARGET_PRODUCT),cm_i925)
```
Comment (JustArchi) [inline boardconfig/BoardConfig.mk:421]: It should be ifeq, not ifneq

**Comment by PhilZ-cwm6** [comment]
agree + there is already a Note 10.1 2014 entry!

**Comment by nicklovell23** [comment]
I could change it to ifeq, and this is not the 2014 note 10.1 but maybe they use the same recovery I could test to find out.

**Comment by PhilZ-cwm6** [comment]
And what about this one:
https://github.com/PhilZ-cwm6/philz_touch_cwm6/blob/cm-11.0/boardconfig/BoardConfig.mk#L390

They are not same?

**Comment by nicklovell23** [comment]
Not 100% sure, I will research more to find out. Sorry for bothering you and doing this pull request.

**Comment by JustArchi** [comment]
Yep, you should add your variant to other threes as they share common properties like width/height etc.

**Pull request closed without merging**